CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+## [1.6.3] - 2023-04-12
+
+### Added
+
+- Entry API responses now include the `created_at` field (#3975)
+- `spire-server agent list` CLI command now shows if agents can be re-attested and supports `by_can_reattest` filtering (#3880)
```

Comment:
```suggestion
- `spire-server agent` CLI commands and Agent APIs now show if agents can be re-attested and supports `by_can_reattest` filtering (#3880)
```
1. This applies to both `spire-server agent list` and `spire-server agent show` CLI commands. Might be easier to just say more generically that this field is presented by the `spire-server agent` CLI subcommand.
2. Probably worth mentioning that the API also supports the same field